app/repositories/application/GeneralApplicationRepository.scala

```diff
@@ -998,6 +1007,74 @@ class GeneralApplicationMongoRepository(timeZoneService: TimeZoneService)(implic
     collection.update(query, passMarkEvaluation, upsert = false) map { _ => }
   }
 
+  // TODO: get rid of this, it feels wrong and we should also be using the App Status case objects.
+  private def applicationStatus(status: String): BSONDocument = {
```

Comment:
Removed as it's not used - using the applciationStatusBSON function to consistently set the progress, application status and progress status date fields instead.